tailwind.config.cjs

```diff
@@ -6,6 +6,7 @@ module.exports = {
     extend: {
       fontFamily: {
         poppins: ['Poppins'],
+        'dm-sans': ['DM Sans'],
```

Comment:
Because I see that they use this font in Figma design.
![image](https://user-images.githubusercontent.com/1421861/190987955-30e87bef-55a8-4073-b03b-cd83da74a310.png)

Reply:
yes it's true, maybe add the class once on the root element of your component, cause all are `DM Sans`
Note that it's not the definitive fonts, but they might use several font so it's good to distinguish them already.

Thanks

Reply:
According to the last appointment, we are going to use the DM sans as  default font

Reply:
Ok, I will put it on the root element then. Thanks!